classes/mail/variables/IssueEmailVariable.php

```diff
@@ -67,4 +71,33 @@ protected function getIssueUrl(): string
             $this->issue->getBestIssueId()
         );
     }
+
+    protected function getIssueToc(): string
+    {
+        $request = Application::get()->getRequest();
+
+        $submissions = Repo::submission()->getCollector()
+            ->filterByContextIds([$this->getContext()->getId()])
+            ->filterByIssueIds([$this->issue->getId()])
+            ->getMany();
+
+        if($submissions->count() <= 0) {
+            return '';
+        }
+
+        $list = $submissions->map(function ($submission) use ($request) {
+            $publication = $submission->getCurrentPublication();
+            $url = $request->getDispatcher()->url(
+                $request,
+                Application::ROUTE_PAGE,
+                $this->getContext()->getPath(),
+                'article',
+                'view',
+                $publication->getData('urlPath') ?? $submission->getId()
+            );
+            return '<li><a href="' . $url . '">' . $publication->getLocalizedFullTitle(null, 'html') . '</a></li>';
```

Comment:
I think the title will still need to get passed through the safe HTML filter, to ensure malicious HTML isn't passed through directly.